source/docs/pipeline_and_webpack.md

```diff
@@ -1,32 +1,48 @@
-title: Asset Pipeline & Webpack
+title: Building & Asset Pipeline
 ---
 
-### Specifying a mode
+As discussed in [Running Apps](running_apps.html#Using-the-run-command), Embark takes care of quite a few things developing applications, including compiling Smart Contracts, JavaScript and other assets. In this guide we'll learn how to take full advantage of Embark's flexibility to build our Smart Contracts or even replace the entire build pipeline for our application's assets.
 
-Embark has by default two modes for the building assets:
+## Building your app
 
-* `development` - Is faster to build, produces sourcemaps, but also outputs bigger js filesizes. Is the default mode of `embark run` and `embark console`
-* `production` - Is slower to build but also outputs much smaller js filesizes. Is the default mode of `embark build` and `embark upload`
+Embark's `build` command enables us to build our decentralized application and deploy it accordingly. Similar to `embark run`, it compiles our Smart Contracts, deploys them to a blockchain network that our app is connected to, processes our app's assets and uploads them to the configured decentralized storage (e.g. IPFS).
 
-You can also specify a particular mode with `--pipeline`, for e.g
+By default, `embark build` will use the `production` mode to build our application.
 
-<pre><code class="shell">$ embark run --pipeline production</code></pre>
+<pre><code class="shell">$ embark build</code></pre>
 
-### Default webpack config
+### Specifying a mode using `--pipeline`
 
-Embark has a internal default webpack config, this config allows you out of the box to:
+Embark comes with two modes for building our application:
 
-* ES6 module `import` and `export` syntax
-* Importing CSS, SCSS, PNG, SVG & Fonts in JS
+* **development** - This mode is for development purposes and produces sourcemaps as well as unoptimized client-side code. Whenever we use `embark run` or `embark console`, this mode is used by default. Since this mode skips code optimizations, it's also the faster mode.
+* **production** - Unsurprisingly, this mode is used to create production artifacts of our application. It produces optimized code and is therefore slower as well. However, client-side code will be highly optimized and therefore smaller in file size This mode is the default for `embark build` and `embark upload` commands.
```

Comment:
```suggestion
* **production** - Unsurprisingly, this mode is used to create production artifacts of our application. It produces optimized code and therefore, takes longer to build. However, client-side code will be highly optimized, thus smaller in file size. This mode is the default for `embark build` and `embark upload` commands.
```
Changed slower because it could sound as the code became slower.
Changed the second "therefore", because synonyms